nodes/analyzer/raycaster_lite.py

```diff
@@ -40,6 +41,7 @@ def sv_init(self, context):
 
         si('SvVerticesSocket', 'Verts')
         si('SvStringsSocket', 'Faces')
+        si('SvStringsSocket', 'MaxDistance').prop_name = 'MaxDistance' #添加输入最大距离接口
```

Comment:
This name also should be fixed

Reply:
There are some errors and I need to close the merge again